internal/kube_helper.go

```diff
@@ -0,0 +1,169 @@
+package internal
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	ctrlRuntime "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/discovery"
+	client "k8s.io/client-go/kubernetes"
+
+	// To authenticate kubeEnv on gke cluster
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
```suggestion
	_ "k8s.io/client-go/plugin/pkg/client/auth"
```
Need to add auth for all..

Reply:
added this change, wherever kubeEnv funcs are imported.